### PR TITLE
Rough Outline for Argo Dashboard

### PR DESCRIPTION
**Pull request by wajiha-oicr**
**Description of changes**

Removed the temp placeholder for the dashboard page. Built an **initial rough outline** for the overall layout of the cards. Separated each card on the page into its own component, laid out the boxes into the row/column orientation. 

Please let me know if things could be implemented better. Still getting used to Typescript and React 😅.

Will continue to work on adding all the features and styling them accordingly.

Screen Shot 2020-01-14 at 1 41 48 PM

**Type of Change**

- [ ] Bug
- [x] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/submission-system/program-dashboard/index.tsx [hunk 1]
```diff
@@ -7,6 +7,16 @@ import usePageContext from 'global/hooks/usePageContext';
 import Banner, { BANNER_VARIANTS } from 'uikit/notifications/Banner';
 import { JUST_JOINED_PROGRAM_STORAGE_KEY } from '../join/details';
 import { SchemaInvalidSubmisisonNotification } from '../SchemaInvalidSubmissionNotification';
+import { Row, Col, ScreenClassRender } from 'react-grid-system';
+import StatsBar from './StatsBar';
+import DonorReleaseSummary from './DonorReleaseSummary';
+import CompletedClinicalData from './CompletedClinicalData';
+import MolecularDataSummary from './MolecularDataSummary';
+import ProgramWorkspaceStatus from './ProgramWorkspaceStatus';
+import DonorDataSummary from './DonorDataSummary';
```
Comment (hlminh2000) [inline components/pages/submission-system/program-dashboard/index.tsx]: these new files seem to be missing from commit :P

Comment (wajiha-oicr) [reply components/pages/submission-system/program-dashboard/index.tsx]: Woops, that's right. They should be added now

File: components/pages/submission-system/program-dashboard/index.tsx [hunk 2]
```diff
@@ -61,14 +79,63 @@ export default function ProgramDashboard() {
           programShortName={programShortName as string}
         />
       }
-      <ContentBox
+      <Row
+        css={css`
+          padding-bottom: 7px;
```
Comment (hlminh2000) [inline components/pages/submission-system/program-dashboard/index.tsx]: I usually just make these closest multiples of 8 :P 

could try something like: 
```
PaddedRow = styled(Row)`padding-bottom: 8px;`
``` 
and use that around (can be done later, up to you :))

File: static/clinical.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="113" height="86" viewBox="0 0 113 86">
```
[the remark this answers is not in the document]
Comment (wajiha-oicr) [reply static/clinical.svg]: I thought so too, but they're a little bit different. The existing assets have this background drawing behind them, and these ones don't.

[the remark this answers is not in the document]
Comment (hlminh2000) [reply static/clinical.svg]: ah I see I see! thanks! :)

[the remark this answers is not in the document]
Comment (wajiha-oicr) [reply static/clinical.svg]: image

File: components/pages/submission-system/program-dashboard/index.tsx (hunk 2 above)
Comment (wajiha-oicr) [reply components/pages/submission-system/program-dashboard/index.tsx]: This makes sense, I'll use this implementation  👍🏽

**Pull request merged**